inc/blocks/namespace.php

```diff
@@ -73,11 +73,18 @@ function on_save_post( int $post_ID, WP_Post $post, bool $update ) : void {
 		return;
 	}
 
-	if ( $post->post_type === POST_TYPE ) {
+	$post_types = array(
+		POST_TYPE,
+		'revision',
+		'customize_changeset'
+	);
```

Comment:
Short array syntax:
```suggestion
	$post_types = [
		POST_TYPE,
		'revision',
		'customize_changeset'
	];
```
I'd probably also call the variable something more explanatory like `$excluded_post_types`, wouldn't hurt to add a filter for that too.